TeamPage.html

```diff
@@ -593,6 +593,48 @@ <h2>Meet Our Team</h2><br>
                </div>
            </div>
           </div>
+          <div class="col-sm-3 col-xs" style="min-width: 210px;">
+            <div class="id_presentation">
+              <img class="img_staff" src="https://i.imgur.com/jxzYjQO.jpg" style="width: 200px; height: 200px; object-fit: cover; object-position: top;" alt="Michelle's Picture"/>
+               <div class="text_pres">Michelle McGowan<br>
+              <div class="subtext_pres">Student Programmer</div>
+               </div>
+               <div class="container" style="text-align: center;">
+                <button type="button" style="margin-top: 10px;" class="btn btn-danger btn-sm" data-toggle="modal" data-target="#myModalMichelleM" onclick="openBSModal('myModalMichelleM', this)">Michelle's Profile</button>
+                <div class="modal fade" tabindex="-1" id="myModalMichelleM" role="dialog" datim-modal>
+                  <div class="modal-dialog">
+                    <div class="modal-content">
+                      <div class="modal-header">
+                        <div class="modal-title"><p class="text-justify"><b>Michelle McGowan</b> - Student Programmer<br>
+                          Work Location: Remote<br>
+                          Email: michelle.mcgowan@unlv.edu</p>
+                        </div>
+                        <button type="button" class="close" data-dismiss="modal" aria-label="close">x</button>
+                      </div>
+                      <div class="modal-body">
+                        <p class="text-justify">Michelle McGowan joined the UNLV-FIA group as a student intern in January 2023. She is currently pursuing a Bachelor of Science degree in Computer Science and hopes to graduate in the Spring of 2024. Michelle will be making sure that Section 508 standards are met on the DATIM site. In her spare time, Michelle enjoys reading, hanging out with friends and going to concerts.</p>

```

Comment:
I believe your final sentence should contain a comma after "friends":
```
In her spare time, Michelle enjoys reading, hanging out with friends(,) and going to concerts.
```
But I could definitely be wrong about that, so please ignore this if so.

Reply:
Thank you! You are right!